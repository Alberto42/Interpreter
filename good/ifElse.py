x = 0
y = 0
if True {
    x = 1
} else {
    x = 2
}

if False {
    y = 1
} else {
    y = 2
}
