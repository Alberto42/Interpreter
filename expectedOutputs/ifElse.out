Ident "y" = 2
Ident "x" = 1


